File: go/1035.go
package main

import "fmt"

func main() {
	var a, b, c, d int
	fmt.Scanf("%d", &a)
	fmt.Scanf("%d", &b)
	fmt.Scanf("%d", &c)
	fmt.Scanf("%d", &d)
	if b > c && d > a && c+d > a+b && c > 0 && d > 0 && a%2 == 0 {
		fmt.Printf("Valores aceitos\n")
	} else {
		fmt.Printf("Valores nao aceitos\n")
	}
}

File: python/1035.py
valores = input().split()

valores = [int(i) for i in valores]

# a = valor0 b = valor1  c = valor2 d = valor3

if (valores[1] > valores[2]) and (valores[3] > valores[0]) and (valores[2] + valores[3] > valores[0] + valores[1]) and (valores[2] > 0) and (valores[3] > 0) and (valores[0] % 2 == 0) :
    print("Valores aceitos")
else:
    print("Valores nao aceitos")